config.ml

```diff
@@ -1,32 +1,28 @@
-(* mirage >= 4.4.1 & < 4.5.0 *)
+(* mirage >= 4.4.0 & < 4.5.0 *)
 (* (c) 2017, 2018 Hannes Mehnert, all rights reserved *)
 
 open Mirage
 
-let keys =
-  let doc = Key.Arg.info ~doc:"nsupdate keys (name:type:value,...)" ["keys"] in
-  Key.(create "keys" Arg.(opt (list string) [] doc))
-
 let dns_handler =
   let packages =
     [
       package "logs" ;
       package ~min:"5.0.0" ~sublibs:["mirage"] "dns-server";
       package "dns-tsig";
     ]
-  and keys = [ Key.v keys ]
+  and runtime_keys = [ runtime_key ~pos:__POS__ "Unikernel.K.keys" ]
```

Comment:
With the latest patches, a typo here (for instance `Unikernel.keys` instead of `Unikernel.K.keys`) will result in the following error message:
```
$ dune build
File "config.ml", line 13, characters 43-57:
Error: Unbound value Unikernel.keys
```